test/gtest/uct/v2/test_uct_query.cc

```diff
@@ -36,11 +35,13 @@ UCS_TEST_P(test_uct_query, query_perf)
                                                            &perf_attr);
     EXPECT_EQ(status, UCS_OK);
 
-    status = uct_iface_query(sender().iface(), &iface_attr);
-    EXPECT_EQ(status, UCS_OK);
-    EXPECT_EQ(iface_attr.bandwidth.dedicated, perf_attr.bandwidth.dedicated);
-    EXPECT_EQ(iface_attr.bandwidth.shared, perf_attr.bandwidth.shared);
-    EXPECT_EQ(iface_attr.overhead, perf_attr.overhead);
+    perf_attr.remote_memory_type = UCS_MEMORY_TYPE_CUDA;
+    perf_attr.operation          = UCT_OP_PUT_SHORT;
+    status                       = uct_iface_estimate_perf(sender().iface(),
+                                                           &perf_attr);
+
+    /* At least one type of bandwidth must be non-zero */
+    EXPECT_NE(0, perf_attr.bandwidth.shared || perf_attr.bandwidth.dedicated);
```

Comment:
Is there anything more precise we can check here?
We can't check only `shared` or `dedicated` - otherwise, tests with some ifaces would fail.
Do we want to differentiate by iface type here and check explicitly that the numbers match?

Reply:
let's check shared+dedicated > epsilon

Reply:
also let's check BW is not same for put/get for cuda and gdrcopy

Reply:
perf_attr.bandwidth.shared + dedicated